.github/workflows/issue-pr-triage.yml

```diff
@@ -3,19 +3,22 @@ name: Auto-triage to project boards
 on:
   issues:
     types: [opened, labeled]
-  pull_request:
+  pull_request_target:
     types: [opened, labeled]
 
 env:
-  GITHUB_TOKEN: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
   content_project:
     runs-on: ubuntu-latest
     name: Triage to Content Project
     steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
I'm a little worried about the checkout here. when using the `pull_request_target` event they warn: 

> Although the workflow runs in the context of the base of the pull request, you should make sure that you do not check out, build, or run untrusted code from the pull request with this event

I just want to clarify what the code getting checked out is, is it the target of the PR and therefore our code, or is it the fork?

Reply:
i assumed it is our code, but to be 100% sure, made it explicit with the repository argument.